src/adapter/src/explain_new/mir/text.rs

```diff
@@ -294,6 +300,12 @@ impl<'a> Displayable<'a, MirRelationExpr> {
                     Ok(())
                 })?;
             }
+            Join {
+                implementation: JoinImplementation::IndexedFilter(id, _key, literal_constraints),
+                ..
+            } => {
+                Self::fmt_indexed_filter(f, ctx, id, Some(literal_constraints.clone()))?;
```

Comment:
`IndexedFilter` doesn't have a child, because it's directly on top of a `Get`, and the `Get` is not shown. For example:
```
Explained Query:
  CrossJoin type=differential // { arity: 3 }
    implementation
      %1 » %0[]
    ArrangeBy keys=[[]] // { arity: 2 }
      Project (#0, #1) // { arity: 2 }
        ReadExistingIndex materialize.public.t1 lookup value (2, "l2")
    Project (#2) // { arity: 1 }
      ReadExistingIndex materialize.public.t2 lookup value (2)
```
So, I think `WITH(linear_chains)` doesn't have to affect it, unless I'm missing something.

Reply:
How does the query from above look like if you add `WITH(linear_chains)`?

Reply:
Looks good.